.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "mcnp2cad"]
+	path = mcnp2cad
+	url = https://github.com/svalinn/mcnp2cad
+	branch = master
```

Comment:
I don't think we want to peg to `master`.  I think we should peg to some specific commit that we know is working and rely on maintainers to update that commit has if/as appropriate.

Reply:
I removed the reference to the `master` branch. I think that what you are suggesting is how submodules work by default: they always point to a specific commit. Right now that commit is `3bb6833b35a45187ecaec100e64cad6313e2998c` which refers to the latest commit in the `master` branch. Once the `mcnp2cad` PR is approved, this commit hash should be changed in this repo to reflect that.